dashboard/src/app/workspaces/workspace-details/workspace-machine-servers/edit-machine-server-dialog/edit-server-dialog.html

```diff
@@ -5,6 +5,7 @@
       <!-- Reference -->
       <che-input-box che-label-name="Reference"
                      che-form="editServerForm"
+                     id="new-server-reference"
```

Comment:
'edit-server-dialog'
It is could be in edit mode. So Id name which starts from 'new-*' isn't correct